dags/resources/stages/parse/table_definitions/zeroex/exchange_v1_0_event_LogCancel.json

```diff
@@ -0,0 +1,107 @@
+{
+    "parser": {
+        "type": "log",
+        "contract_address": "0x12459c951127e0c374ff9105dda097662a027093",
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "name": "maker",
+                    "type": "address"
+                },
+                {
+                    "indexed": true,
+                    "name": "feeRecipient",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "makerToken",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "takerToken",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "cancelledMakerTokenAmount",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "cancelledTakerTokenAmount",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": true,
+                    "name": "tokens",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "name": "orderHash",
+                    "type": "bytes32"
+                }
+            ],
+            "name": "LogCancel",
+            "type": "event"
+        },
+        "field_mapping": {}
+    },
+    "table": {
+        "project_name": "blockchain-etl",
+        "dataset_name": "zeroex",
+        "table_name": "exchange_v1_0_event_LogCancel",
+        "table_description": "Lorem ipsum.",
```

Comment:
If possible please provide short table descriptions for each event.

Reply:
Done for 0x using https://0x.org/docs/guides/v2-specification

Reply:
Done for ENS using https://github.com/ensdomains/ens/blob/master/contracts/ENS.lll